CONTRIBUTING.MD

```diff
@@ -0,0 +1,186 @@
+<p align="center">
+  <a href="https://www.algolia.com">
+      <img alt="Algolia for Ruby" src="https://raw.githubusercontent.com/algolia/algoliasearch-client-common/master/banners/ruby.png" >
+    </a>
+</p>
+
+Hello and welcome to the contributing guide for algolia gem. Thanks for considering participating in our project 🙇
+
+If this guide does not contain what you are looking for and thus prevents you from contributing, don't hesitate to leave a message on the [community forum](https://discourse.algolia.com/) or to [open an issue](https://github.com/algolia/instantsearch.js/issues).
+
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN TocDown TO UPDATE -->
+<!-- TocDown Begin -->
+* [Reporting an issue](#reporting-an-issue)
+* [The code contribution process](#the-code-contribution-process)
+* [Commit conventions](#commit-conventions)
+* [Branch organization](#branch-organization)
+* [Requirements](#requirements)
+* [Code structure](#code-structure)
+  * [The source algolia folder](#the-source-algolia-folder)
+* [Tests](#tests)
+* [Linting](#linting)
+<!-- TocDown End -->
+
+## Reporting an issue
+
+Opening an issue is very effective way to contribute because many users might also be impacted. We'll make sure to fix it quickly if it's technically feasible and doesn't have important side effects for other users.
```

Comment:
```suggestion
Opening an issue is very effective way to contribute because other users might also be impacted. We'll make sure to fix it quickly if it's technically feasible and doesn't have important side effects for other users.
```